README.md

```diff
@@ -35,15 +35,18 @@ See [the pip documentation] if you need to learn more about pip.
 
 ```
 cd
+become
+forks
 list
 list groups
 serial
+!  - force shell module instead of the ansible module of the same name (!yum update -y)
 ```
 
 ## Usage:
 
 ```
-[root@aaa ~]# ansible-shell
+[root@aaa ~]# ansible-shell --one-line
```

Comment:
That exact output format is from the very first release and it's unfortunatley no longer available but you are right `--one-line` is working, I wasn't aware of that. that's pretty cool :+1: